### ISSUE TITLE
Setup Channel routing

### ISSUE THREAD

**Issue by 0xTranqui** [1ifeworld/river#88]
tldr is we need to do a bit of prep work to enable routing to specific channels. right now the ways we can distinguish channels are nodes that were registered with the specific channel schema hash, and then whatever the node id is. to future proof to a world with a node registry migration to another chain, we probably want to use some type of hash scheme over all the data so that the routing for channels can work across node registry deploys

proposal: river.ph/channel/{kecack256(abi.encode(chainId/nodeRegistryAddress/channelSchema/nodeId))}

scope:
- [x]  add this the creation of this hash to the Node register crud, as well as updating the underlying entity (Node and/or Channel?)
- [x] create a new route in the app that allows someone to access a specific channel by entering the unique hash
